genie-web/src/main/resources/application.yml

```diff
@@ -109,6 +109,9 @@ genie:
     scheduler:
       pool:
         size: 5
+  grpc:
+    server:
+      enabled: true
```

Comment:
Also we need to document these properties. (I think I have a TODO to do that for some of the job file stuff I did too)

Reply:
Integration tests are failing currently, and it's due to this.
I think it does not shutdown cleanly after bringing up the server and so at the next start it throws `BindException` (due to port already in use).
So for example, I'm going to disable gRPC server autostart in integration tests, since we don't use it anyway.
I'll document the properties.